src/docfx/cli/Program.cs

```diff
@@ -59,6 +59,10 @@ internal static async Task<int> Run(string[] args)
                             await Build.Run(docset, options, report);
                             Done(stopwatch.Elapsed, report);
                             break;
+                        case "gc":
+                            await GarbageCollector.Collect(options.NotAccessedDays);
+                            Done(stopwatch.Elapsed, report);
```

Comment:
gc result is a bit different from build, restore. It does not have error/warning count.

Can we instead show how may files are removed from download cache, how many git repo checkouts have been removed?